### DISCUSSION TITLE
Headless LTSP server: jobs running at clients are immediately  terminated uppon exiting  from terminal.

### DISCUSSION THREAD

**Discussion by abdeoliveira** [ltsp/ltsp#885] [Q&A] [answered]
I have this headless LTSP server (chrootless) and 10 PXE clients we mounted at my university to perform computational calculations on the clients. The basic _modus operandi_ is as follows:

1. login to the server.
2. ssh to a free client.
3. submit some jobs like `./exec > output &`
4. exit from terminal.

It happens that jobs are immediately killed at `exit` from terminal. 
After playing with the `/etc/ltsp/ltsp.conf` file I found this odd combination of parameters which 
happen to **keep my jobs running at clients after logout**:

```
[server]
#NFS_HOME=1

[clients]
FSTAB_HOME="server:/home /home nfs defaults,nolock 0 0"
```

If I understand right, this combination of parameters won't make much sense (note `NFS_HOME` is commented) 
and I would like to understand what's going on. 

Any help would be much appreciated.

**Comment by alkisg** [comment]
Sorry not enough time for a lengthy post, but here are two similar topics to get you started:

https://github.com/ltsp/ltsp/discussions/235
https://github.com/ltsp/ltsp/discussions/398

**Reply by abdeoliveira** [reply, marked as answer]
Thank you **very** much. Both posts were essential for me to understand what is happening. 

I understand that changing `KillUserProcesses=yes` to `=no` may cause issues and etc. But in our case (headless server with bare minimum set of applications and very specific use case) I believe this will not cause any problems. 

I adopted your suggestion, i.e., adding

```
POST_INIT_LOGIND="sed 's/.*\(KillUserProcesses=\).*/\1no/' -i /etc/systemd/logind.conf"
```
in `ltsp.conf` just worked like a charm.
